plugin/eureka/setup.go

```diff
@@ -0,0 +1,128 @@
+package eureka
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/coredns/coredns/core/dnsserver"
+	"github.com/coredns/coredns/plugin"
+	"github.com/coredns/coredns/plugin/pkg/fall"
+	clog "github.com/coredns/coredns/plugin/pkg/log"
+
+	"github.com/caddyserver/caddy"
+)
+
+var log = clog.NewWithPlugin("eureka")
+
+var f = func(baseUrl string) clientAPI {
+	return client{
+		BaseUrl: baseUrl,
+	}
+}
+
+func init() {
+	plugin.Register("eureka", setup)
+}
+
+func setup(c *caddy.Controller) error {
+	for c.Next() {
+		zones := c.RemainingArgs()
+
+		var fall fall.F
+		var eurekaBaseUrl string
+
+		eurekaOptions := &options{
+			refresh: time.Duration(1) * time.Minute, // default update frequency to 1 minute
+			ttl:     30,                             // default TTL to 30 seconds
```

Comment:
Why `30`? "Usual" default is 300. The comment should explain why it is set to 30 instead of what it is set to (already clear from the code).